eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -133,7 +133,7 @@ steps:
     solution: "build\\build.proj"
     configuration: "$(BuildConfiguration)"
     msbuildArguments: "/restore:false /target:CoreUnitTests;UnitTestsVS /property:BuildRTM=$(BuildRTM) /property:BuildNumber=$(BuildNumber) /property:SkipILMergeOfNuGetExe=true /binarylogger:$(Build.StagingDirectory)\\binlog\\07.RunUnitTests.binlog"
-  condition: "and(succeeded(), eq(variables['BuildRTM'], 'true'), not(eq(variables['IsOfficialBuild'], 'true')))"
+  condition: "and(succeeded(), eq(variables['BuildRTM'], 'true'), not(eq(variables['IsOfficialBuild'], 'true')), not(eq(variables['RunUnitTestsOnWindows'], 'false')))"
```

Comment:
The changes to oficial.yml change a parameter, and this is reading a variable, which is not the same thing. Looking at pipeline.yml, it doesn't appear to me that the parameter value is used/passed through.  I think additional changes might be needed?

Reply:
oops, I missed that official.yml copies the parameter value to a variable.

But if we expect the tests to run in the public CI pipieline, why don't we start deleting all these tests from the official & private PR builds instead?